src/storage.ts

```diff
@@ -17,6 +25,13 @@ const V0_DEDUP_RTV_PREFIXES: ReadonlySet<string> = new Set([
   '24',
 ]);
 
+/**
+ * Whether the client supports Brotli compression.
+ */
+function tryBrotli(cf?: IncomingCloudflarePropertiesExtended): boolean {
+  return /\bbr\b/.test(cf?.clientAcceptEncoding ?? '');
```

Comment:
Nit:
```suggestion
  return /(^|,)\s*br\s*(,|;|$)/.test(cf?.clientAcceptEncoding ?? '');
```

Reply:
It actually parses the quality list, eg a `br-2` encoding shouldn't trigger the brotli format.

Reply:
Is that a real concern though? `br2` (assuming that ever comes, and the formatting mirrors e.g., `bzip2`) wouldn't trigger Brotli, and this makes the code much nicer to read

Reply:
This is a nit, so it's up to you. This parsing is incorrect per the spec, which is the reason I suggested the fix.

Reply:
I think I'd rather leave it as-is then, simple regex > complex for-only-being-technically-correct regex 😅